controllers/skipjob/egress_service_entry.go

```diff
@@ -0,0 +1,54 @@
+package skipjobcontroller
+
+import (
+	"context"
+	"fmt"
+	skiperatorv1alpha1 "github.com/kartverket/skiperator/api/v1alpha1"
+	"github.com/kartverket/skiperator/pkg/resourcegenerator/istio"
+	"github.com/kartverket/skiperator/pkg/util"
+	networkingv1beta1 "istio.io/client-go/pkg/apis/networking/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	ctrlutil "sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func (r *SKIPJobReconciler) reconcileEgressServiceEntry(ctx context.Context, skipJob *skiperatorv1alpha1.SKIPJob) (reconcile.Result, error) {
```

Comment:
Did reusing the reconciliation logic from Application get too messy?

Reply:
I believe it to be possible, but yeah its somewhat messy due to some few differences in how we handle stuff. There's definitely more refactoring one could do to make it cleaner, but this is already a big PR 😮‍💨

Reply:
No problem, I'm personally not a DRY person but wanted to check. Readability trumps DRY. 

If a simple abstraction is possible here we can look at it in a later PR.